examples/transmitter/swagger_server/swagger/swagger.yaml

```diff
@@ -840,6 +860,31 @@ components:
             Stream Management API calls that require authorization.
       example:
         token: 49e5e7785e4e4f688aa49e2585970370
+    TriggerEventParameters:
+      title: Trigger Event Parameters
+      required:
+      - event_type
+      - subject
+      type: object
+      properties:
+        event_type:
+          type: string
+          description: "Read-Write.\nSupports all  (not yet supported)[RISC](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html)\
+            \ and (supported)[CAEP](https://openid.net/specs/openid-caep-specification-1_0-ID1.html)\
+            \ event types.      "
+          example: credential-compromise
+        subject:
+          allOf:
+          - $ref: '#/components/schemas/Subject'
+          - description: REQUIRED. SHOULD be same as subjects receiver cares about
```

Comment:
We should add a comment that says we only support Email format.

Reply:
If that's the case, is it sufficient to mention in the comments or should we create a separate definition?